changelog/@unreleased/pr-791.v2.yml

```diff
@@ -0,0 +1,8 @@
+type: improvement
+improvement:
```

Comment:
we should flag [this as a break](https://github.com/palantir/gradle-baseline/pull/493#issuecomment-451897295) so any folks lagging on gradle < 5.x upgrade

Reply:
As I mentioned in the pr description it's possible to not make it a break so I leave it to the maintainers to decide what they prefer